distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/SystemStateManager.java

```diff
@@ -172,6 +172,12 @@ private void resetSystem() {
       // reset the catalog
       console.runCommand("catalog:removeall -f -p");
       console.runCommand("catalog:removeall -f -p --cache");
+
+      // Restart bundles that manage dynamic metacards
+      // TODO: 1/19/18 DDF-3561 remove the plugin bundle from this list
+      serviceManager.restartBundles(
+          "registry-identification-plugin", "registry-federation-admin-service-impl");
```

Comment:
✏️  Should we note that order matters?

Reply:
I don't think it's necessary. Honestly, Karaf is going to do its thing and short of adding in logic to check the state of each bundle after the stop and start steps of the restart before moving forward, it's possible that they'll start in different order anyway. The key is to clear out the cached value.